[IMP] sales: mass cancel quotation/SOs (#10097)
odoo/documentation: content/applications/sales/sales/send_quotations/quote_template.rst

return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
back-end of the *Sales* application.

Mass cancel quotations/sales orders
===================================

Cancel multiple quotations (or sales orders) by navigating to the :menuselection:`Sales app -->
Orders --> Quotations` dashboard, landing, by default, in the list view. Then, on the left side of
the table, tick the preferred checkboxes for removal.

> .. tip::
>   Select all records in the table by selecting the checkbox column header at the top-left of the
>   table; the total number of selected items are displayed at the top of the page.

Reviewer: Indent admonition text :)
```suggestion
.. tip::
   Select all records in the table by selecting the checkbox column header at the top-left of the
   table; the total number of selected items are displayed at the top of the page.
```

Then, with the desired quotations (or sales orders) selected from the list view on the
:guilabel:`Quotations` page, click the :icon:`fa-cog` :guilabel:`Actions` button to reveal a
drop-down menu.

From this drop-down menu, select :guilabel:`Cancel quotations`.

.. image:: quote_template/cancel-quotations.png
   :align: center
   :alt: The Cancel quotations option on the Actions drop-down menu in the Odoo Sales application.

.. note::
   This action can be performed for quotations in *any* stage, even if it is confirmed as a sales
   order.

Upon selecting the :guilabel:`Cancel quotations` option, a :guilabel:`Cancel quotations`
confirmation pop-up window appears. To complete the cancellation, click the :guilabel:`Cancel
quotations` button.

.. note::
   An error pop-up message appears when attempting to cancel an order for an ongoing subscription
   that has an invoice.

>    .. image:: quote_template/validation-error.png
>       :align: center
>       :alt: The Cancel quotations option on the Actions drop-down menu in the Odoo Sales application.
> 

Reviewer: IMO this image is unnecessary. If you decide to keep it, please give it a unique alt tag :) 
```suggestion
```

return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
back-end of the *Sales* application.

Mass cancel quotations/sales orders
===================================

> Cancel multiple quotations (or sales orders) at once from the list view of the
> :guilabel:`Quotations` page.
> 
> First navigate to the :guilabel:`Quotations` page by going to :menuselection:`Sales app
> --> Orders --> Quotations`, or open the **Sales** app from the main Odoo dashboard.
> 
> The :guilabel:`Quotations` page displays all the quotations and sales order in a default list view.
> 
> .. image:: quote_template/quotations-page.png
>    :align: center
>    :alt: The main Quotations page in the Odoo Sales application, in a default list view.
> 
> From here, select the desired quotations (or sales orders) from the list, by ticking the checkbox
> on the far-left of their respective line. To select *all* the items on the list, tick the checkbox
> to the far-left of the column titles, and all the items become selected.
> 
> The total number of selected items are displayed at the top of the page.

Reviewer 2: Can be more succinct: 17 lines --> 7
Simple instructions do not need so much breadth. 

```suggestion
Cancel multiple quotations (or sales orders) by navigating to the :menuselection:`Sales app -->
Orders --> Quotations` dashboard, landing, by default, in the list view. Then on the left side of
the table, tick the preferred checkboxes for removal.

..tip::
  Select all records in the table by selecting the checkbox column header at the top-left of the
  table; the total number of selected items are displayed at the top of the page.
```

.. image:: quote_template/quotations-selected.png
   :align: center
   :alt: Mass selections made on the Quotations page in the Odoo Sales application.

Then, with the desired quotations (or sales orders) selected from the list view on the
:guilabel:`Quotations` page, click the :icon:`fa-cog` :guilabel:`Actions` button to reveal a
drop-down menu.

From this drop-down menu, select :guilabel:`Cancel quotations`.

> .. image:: quote_template/cancel-quotations.png

Reviewer 2: This is the only screenshot you need out of the three since it shows the column header (select all), action menu, and # selected,.

return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation form in the
back-end of the *Sales* application.

Mass cancel quotations/sales orders
===================================

Odoo provides the ability to cancel multiple quotations (or sales orders) at once from the list view
of the :guilabel:`Quotations` page (the main *Sales* app dashboard).

To do so, first navigate to the :guilabel:`Quotations` page by going to :menuselection:`Sales app
--> Orders --> Quotations`, or open the *Sales* app from the main Odoo dashboard.

Once on the :guilabel:`Quotations` page, all the quotations and sales order are presented in a
default list view.

> .. image:: quote_template/quotations-page.png

Reviewer 3: I believe this image needs to be shrunk down to 760px or lower since it's currently 930px wide.

Author: So, ideally, you want to aim for something in the 760px range, but the true limit is 933px... so, taking that into consideration, I'm going to opt to keep these screenshots as-is, but if any other reviewer suggests the same, I'll be sure to implement the change (and give you 100% credit for the original suggestion 👍 )

   :align: center
   :alt: The main Quotations page in the Odoo Sales application, in a default list view.

From here, select the desired quotations (or sales orders) from the list, by ticking the checkbox
on the far-left of their respective line. To select *all* the items on the list, tick the checkbox
to the far-left of the column titles, and all the items become selected.

The total number of selected items are displayed at the top of the page.

> .. image:: quote_template/quotations-selected.png

Reviewer 3: Same as above.

   :align: center
   :alt: Mass selections made on the Quotations page in the Odoo Sales application.

Then, with the desired quotations (or sales orders) selected from the list view on the
:guilabel:`Quotations` page, click the :icon:`fa-cog` :guilabel:`Actions` button to reveal a
drop-down menu.

From this drop-down menu, select :guilabel:`Cancel quotations`.

> .. image:: quote_template/cancel-quotations.png

Reviewer 3: Same as above.

   :align: center
   :alt: The Cancel quotations option on the Actions drop-down menu in the Odoo Sales application.

.. note::
   This action can be performed for quotations in *any* stage, even if it is confirmed as a sales
   order.

Upon selecting the :guilabel:`Cancel quotations` option, Odoo presents a :guilabel:`Cancel
quotations` confirmation pop-up window. To complete the cancellation, click the :guilabel:`Cancel
quotations` button. To cancel, click the :guilabel:`Discard` button.

.. note::
   Odoo provides an error pop-up message when attempting to cancel an order for an ongoing
   subscription that has an invoice.

>    .. image:: quote_template/validation-error.png

Reviewer 3: Same as above.
